src/pkg/reg/model/policy.go

```diff
@@ -15,7 +15,6 @@
 package model
 
 import "github.com/goharbor/harbor/src/lib/errors"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380